### DISCUSSION TITLE
Volunteer to moderate issues

### DISCUSSION THREAD

**Discussion by alandtse** [alandtse/tesla#726] [General]
Let me know if you'd be up for managing issues particularly the latest powerwall issue. Please let me know what repos if any you've managed too please.

**Comment by skipishere** [comment]
I'd be happy to help out where/when I can. I'm a daily user of GitHub at work (mostly private repos so can't show much for it).

**Reply by alandtse** [reply]
Awesome. I'll invite you as a contributor. I believe that should give you the ability to moderate issues; but let me know if you don't seem to be able to.

**Reply by skipishere** [reply]
Looks like I have the ability now, as well as `push` permission (wasn't sure if you meant for push as well?)

**Reply by alandtse** [reply]
I don't think I have the granularity to disable the push. Just don't let the power corrupt you. ;)

And thanks for helping managing things. Feel free to @ me if there's something you need my view on.
